packages/devtools_app/lib/src/config_specific/server/_server_web.dart

```diff
@@ -250,6 +250,49 @@ Future<void> resetDevToolsFile() async {
   }
 }
 
+Future<DevToolsJsonFile> requestPrimaryAppSizeFile(String path) async {
+  if (isDevToolsServerAvailable) {
+    final resp = await _request(
+      '$apiGetPrimaryAppSizeFile?$primaryAppSizeFilePropertyName=$path',
+    );
+    if (resp?.status == HttpStatus.ok) {
+      return _devToolsJsonFileFromAppSizeResponse(resp, path);
+    } else {
+      logWarning(resp, apiGetPrimaryAppSizeFile);
+    }
+  }
+  return null;
+}
+
+Future<DevToolsJsonFile> requestSecondaryAppSizeFile(String path) async {
+  if (isDevToolsServerAvailable) {
+    final resp = await _request(
+      '$apiGetSecondaryAppSizeFile?$secondaryAppSizeFilePropertyName=$path',
+    );
+    if (resp?.status == HttpStatus.ok) {
+      return _devToolsJsonFileFromAppSizeResponse(resp, path);
+    } else {
+      logWarning(resp, apiGetPrimaryAppSizeFile);
+    }
+  }
+  return null;
+}
+
+DevToolsJsonFile _devToolsJsonFileFromAppSizeResponse(
+  HttpRequest resp,
+  String filePath,
+) {
+  final _json = json.decode(resp.response);
```

Comment:
Making a variable private is a little odd. Maybe call this `data` or map instead of `_json`? Also is _json of type dynamic?

Reply:
fixed